CHANGELOG.md

```diff
@@ -45,6 +45,7 @@
 - Dev: Added CMake Install Support on Windows. (#4300)
 - Dev: Changed conan generator to [`CMakeDeps`](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmakedeps.html) and [`CMakeToolchain`](https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmaketoolchain.html). See PR for migration notes. (#4335)
 - Dev: Refactored 7TV EventAPI implementation. (#4342)
+- Dev: Added support for building with crashpad. See PR for build instructions. (#4351)
```

Comment:
If this stays in CI this could be changed into a minor entry.

Reply:
It should stay in CI